### PR TITLE
docs: add example readme

### PR DESCRIPTION
**Pull request by mweibel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,47 @@ $ ./varnish-plus-cli.phar vcl:deploy -u https://$HOST --username $USERNAME  --pa
...
+DIST_DIR ?= dist
+BIN_DIR ?= bin
+TEMPLATE_DIR ?= templates
+
+all: install compile
+
+install: $(BIN_DIR) $(BIN_DIR)/varnish-plus-cli.phar
+
+$(BIN_DIR):
+	mkdir -p $(BIN_DIR)
```
Comment (dbu) [inline README.md:78]: are these actual tabulator?

**Review by dbu — COMMENTED** [review]
great! i forgot how elegant makefiles are

only question is with the tabulator (as makefiles are quite picky on tab vs spaces...)

File: README.md (hunk 1 above)
Comment (mweibel) [reply README.md:78]: they are, but I'm unsure if some processing is done, it certainly looks weird here. 
Anyway I added a note to make it clear.

**Comment by mweibel** [comment]
they indeed are :) 
I added a note about it to ensure people don't copy-paste it wrongly.

**Review by dbu — APPROVED** [review]

**Comment by dbu** [comment]
the note is a good idea anyways :+1:

**Pull request merged**